quickwit/quickwit-janitor/src/actors/delete_task_planner.rs

```diff
@@ -522,7 +523,7 @@ mod tests {
         );
         // Trigger new plan evaluation and check that we don't have new merge operation.
         delete_planner_mailbox
-            .send_message(PlanDeleteOperations)
+            .ask(PlanDeleteOperations)
```

Comment:
send message just queues a message and returns. It does not wait for the message to be processed.